packages/celotool/src/cmds/geth/start.ts

```diff
@@ -0,0 +1,208 @@
+import { GethArgv } from '../geth'
+
+export const command = 'start'
+export const describe = 'command for running geth'
+
+interface StartArgv extends GethArgv {
+  networkId: string
+  syncMode: string
+  mining: boolean
+  blockTime: number
+  nodekeyhex: string
+  minerGasPrice: number
+  port: number
+  rpcport: number
+  wsport: number
+  verbosity: number
+  verbose: boolean
+  instances: number
+  purge: boolean
+  withProxy: boolean
+  ethstats: string
+}
+
+export const builder = (argv: yargs.Argv) => {
+  return addCeloGethMiddleware(argv)
+    .option('network-id', {
+      type: 'string',
+      description: 'network id',
+      default: '1101',
+    })
+    .option('sync-mode', {
+      choices: ['full', 'fast', 'light', 'ultralight'],
+      default: 'full',
+    })
+    .option('mining', {
+      type: 'boolean',
+      description: 'Is mining enabled',
+      default: false,
+    })
+    .option('miner-gas-price', {
+      type: 'number',
+      description: 'Mining gas price',
+      default: 0,
+    })
+    .option('port', {
+      type: 'number',
+      description: 'Port',
+      default: 30303,
+    })
+    .option('rpcport', {
+      type: 'number',
+      description: 'HTTP-RPC server listening port',
+      default: 8545,
+    })
+    .option('wsport', {
+      type: 'number',
+      description: 'WS-RPC server listening port',
+      default: 8546,
+    })
+    .option('instances', {
+      type: 'number',
+      description: 'Number of instances to run',
+      default: 1,
+    })
+    .option('with-proxy', {
+      type: 'boolean',
+      description: 'Start with proxy in front',
+      default: false,
+    })
+    .option('verbosity', {
+      type: 'number',
+      description: 'Geth Verbosity level',
+      default: 5,
+    })
+    .option('verbose', {
+      type: 'boolean',
+      description: 'Command verbosity flag',
+      default: false,
+    })
+    .option('purge', {
+      type: 'boolean',
+      description: 'This will purge the data directory before starting.',
+      default: false,
+    })
+    .option('ethstats', {
+      type: 'string',
+      description: 'address of the ethstats server',
+    })
+    .option('blockTime', {
+      type: 'number',
+      description: 'Block Time',
+      default: 1,
+    })
+}
+
+export const handler = async (argv: StartArgv) => {
+  const verbosity = argv.verbosity
+  const verbose = argv.verbose
+
+  const gethDir = argv.gethDir
+  const datadir = argv.dataDir
+  const networkId = parseInt(argv.networkId, 10)
+  const syncMode = argv.syncMode
+  const blockTime = argv.blockTime
+
+  const port = argv.port
+  const rpcport = argv.rpcport
+  const wsport = argv.wsport
+
+  const mining = argv.mining
+  const minerGasPrice = argv.minerGasPrice
+  const network = 'local'
+  const instances = argv.instances
+
+  const mnemonic =
+    'jazz ripple brown cloth door bridge pen danger deer thumb cable prepare negative library vast'
```

Comment:
Does it make sense to have this hard-coded, rather than using the value defined in `.env.mnemonic`/`.env.mnemonic.$TESTNET_NAME`?